.github/workflows/ci.yml

```diff
@@ -7,15 +7,7 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
-
-      - name: Use Node.js 16 LTS
```

Comment:
Also, I'm not sure that we want to loose the ability to *manually* specify the exact Node.js that we use for testing!?

Reply:
Yes I agree, maybe keeping setup-node can be the soulution is important. I can find for you the source of the information, but if you are intersted to force a specific nodejs version I cannot change your mind.